metadata-ingestion/tests/integration/oracle/test_oracle.py

```diff
@@ -0,0 +1,47 @@
+from unittest import mock
+from unittest.mock import MagicMock
+
+import pytest
+from freezegun import freeze_time
+
+from tests.integration.oracle.common import OracleTestCaseBase  # type: ignore
+
+FROZEN_TIME = "2022-02-03 07:00:00"
+
+
+class OracleIntegrationTestCase(OracleTestCaseBase):
```

Comment:
prefer composition over inheritance

Reply:
We can use self.<method> to call common functions. so it is not a complex hierarchy.